cumulusci.yml

```diff
@@ -48,69 +48,76 @@ tasks:
 
     execute_qa_apex:
-        description: Runs Apex setup for QA orgs.
+        description: Runs anonymous apex for qa to enable course connections.
         class_path: cumulusci.tasks.apex.anon.AnonymousApexTask
         options:
             path: scripts/setup.cls
             apex: qaSetup();
 
     retrieve_config_dev:
-        description: Retrieves the current changes in the scratch org into unpackaged/config/dev
+        description: Retrieves the current changes in the scratch org into unpackaged/config/dev.
         class_path: cumulusci.tasks.salesforce.sourcetracking.RetrieveChanges
         options:
             path: unpackaged/config/dev
             namespace_tokenize: $project_config.project__package__namespace
 
     retrieve_config_qa:
-        description: Retrieves the current changes in the scratch org into unpackaged/config/qa
+        description: Retrieves the current changes in the scratch org into unpackaged/config/qa.
         class_path: cumulusci.tasks.salesforce.sourcetracking.RetrieveChanges
         options:
             path: unpackaged/config/qa
             namespace_tokenize: $project_config.project__package__namespace
 
     retrieve_config_trial:
-        description: Retrieves the current changes in the scratch org into unpackaged/config/trial
+        description: Retrieves the current changes in the scratch org into unpackaged/config/trial.
         class_path: cumulusci.tasks.salesforce.sourcetracking.RetrieveChanges
         options:
             path: unpackaged/config/trial
             namespace_tokenize: $project_config.project__package__namespace
 
     robot:
+        description: Runs a Robot Framework test from a .robot file.
         options:
             suites: robot/EDA/tests
             options:
                 outputdir: robot/EDA/results
 
     robot_libdoc:
+        description: Generates documentation for project keyword files.
         options:
             path: robot/EDA/resources/EDA.py,robot/EDA/resources/EDA.robot
             output: robot/EDA/doc/Keywords.html
 
     update_admin_profile:
+        description: Retrieves, edits, and redeploys the System Administrator profile with full FLS perms for all objects/fields. Also sets record type visibilities.
+        group: Custom Tasks
         class_path: tasks.salesforce.UpdateAdminProfile
 
 flows:
     edaectomy:
-        description: Remove all the EDA metadata in the target Org
+        description: Remove all the EDA metadata in the target Org.
         steps:
             1:
                 task: delete_dev_config
             2:
                 task: uninstall_packaged
 
     config_apextest:
+        description: Configure an org to run apex tests after package metadata is deployed.
         steps:
             3:
                 task: execute_install_apex
 
     config_dev:
+        description: Configure an org for use as a dev org after package metadata is deployed.
```

Comment:
I'm guessing that you're adding descriptions here so that the configuration file is easier to understand. I think that makes sense, but should we consider that by doing so we run the risk of them going out of sync? Perhaps for those built-in tasks/flows that you're adding descriptions for we could add comments instead?

Reply:
That is a great idea!